wiki/Tournaments/SSSS/3/en.md

```diff
@@ -0,0 +1,674 @@
+   - Head-to-head records between the tied teams
+   - Number of wins
+   - Sum of wins of the tied team’s opponents
+   - The result of an extra play-off match (if needed)
+6. Forfeiting a match will be treated as an instant loss with -5 maps difference to the forfeiting team's name.   
+7. The top 16 teams based on the Swiss round standings will advance into the knock-out stage.
+
+#### Knock-out stage rules
+
+1. The 16 teams who managed to get through from the Swiss round will be matched to each other based on the following system:
+   - The top 4 teams in order will be given the first choice to pick their Round of 16 opponent among the 9th-16th place teams.
+   - The 5th-8th placed teams will then be faced against the unchosen teams based on reverse placement order in the Round of 16.
+2. Teams will compete against each other using the double-elimination system.
+3. The double-elimination system works as follows:
+   - Teams who lose in the upper bracket can still play again in the lower bracket.
+   - Teams who lose in the lower bracket will be eliminated from the tournament.
+   - In the Grand Final, the winner of the upper bracket will be given a 2-points advantage against the winner of the lower bracket. **There will be no bracket reset in the tournament**.
+4. Players who can compete in the next round are determined by:
+   - In the the Round of 16, the Quarterfinals, and the Semifinals, each team needs to win 7 points in order to win a match. (Best of 13)
+   - In both of the Finals weeks, excluding the Grand Final match, each team needs to win 8 points in order to win a match. (Best of 15)
+   - In the Grand Final match, each player needs to win 9 points in order to win a match (Best of 17) with the winner of the upper bracket given a 2-points advantage at the start of the match.
+   - Whether there are teams who are declared to win the match by default.
+   - Whether there are teams who are disqualified from the tournament.
+
+### Match regulations
+
+1. Each team captain must use the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
+   - The loser of the `!roll` gets the opposite by default.
+2. Each team has to ban **one beatmap** prior to the start of the match and **one other beatmap** prior to the third round of picks (that are yet to be banned or picked at that point) from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match.
+3. For each map, each team is required to field either **1 Seed B player and 1 Seed C player** or **2 Seed C players** at minimum.
+   - If a team is missing or otherwise not able to ready their lower seed players for the match, they are still allowed to continue playing with one player disadvantage (2v3) or two players disadvantage (1v3) should they wish to.
```

Comment:
```suggestion
   - If a team is missing or otherwise not able to ready their lower seed players for the match, they are still allowed to continue playing at a player disadvantage (2v3 or 1v3) should they wish to.
```